packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -724,3 +728,156 @@ describeNoCompat("SingleCommit Summaries Tests", (getTestObjectProvider) => {
 		);
 	});
 });
+
+describeNoCompat("IdCompressor Summaries", (getTestObjectProvider) => {
+	let provider: ITestObjectProvider;
+	const config: ITestContainerConfig = {
+		...testContainerConfig,
+		runtimeOptions: { ...testContainerConfig.runtimeOptions, enableRuntimeIdCompressor: true },
+	};
+	beforeEach(() => {
+		provider = getTestObjectProvider();
+	});
+
+	it("Summary includes IdCompressor when enabled", async () => {
+		const { mainContainer } = await createMainContainerAndSummarizer(provider, config);
+
+		const defaultDataStore = await requestFluidObject<ITestDataObject>(
+			mainContainer,
+			defaultDataStoreId,
+		);
+		const containerRuntime = defaultDataStore._context.containerRuntime as ContainerRuntime;
+
+		await provider.ensureSynchronized();
+
+		const { summary } = await containerRuntime.summarize({ trackState: false });
```

Comment:
Why keep trackState: false?

Reply:
These tests have been rewritten to use `summarizeNow` instead and check its result. I'm not sure why, but I had to keep `trackState: false` or I got a runtime error.